src/webportal/src/app/job/job-view/fabric/job-event/conn.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { clearToken } from '../../../../user/user-logout/user-logout.component';
```

Comment:
You can use openpaisdk: `jobClient.listEvents(username, jobname, "Warning")` here instead of write the http request manually.

But I'm not sure if it is convenient for you.

Reply:
Here is the reference: https://github.com/microsoft/openpaisdk/blob/6903b18efe6649e715d3d82bbedf0b9ee5de7b1b/tests/unit_tests/jobClient.spec.ts#L200

Reply:
Has it already published to npm registry? I need update openpaisdk version before using.

Reply:
No, the changes are not pushed to npm.